File: Java_Weather_API_Library/src/main/java/command/WeatherCommand.java
package command;

import api.OpenWeatherAPI;

import java.util.List;

public class WeatherCommand implements CommandBase {

    @Override
    public void runCommand(String name) {
        if (name.equalsIgnoreCase("get_weather")) {
            OpenWeatherAPI openWeather = OpenWeatherAPI.getInstance(35.41, 139.41); //東京
            for (OpenWeatherAPI.WeatherList weatherList : openWeather.getWeatherList()) {
                OpenWeatherAPI.WeatherList.Weather weather = weatherList.getWeather().getFirst();
                OpenWeatherAPI.WeatherList.Main weatherMain = weatherList.getMain();
                System.out.print("\n----------" + "\n" +
                        "時間：" + weatherList.getLocalDateTime() + "\n" +
                        "天気: " + weather.getDescription() + "\n" +
                        "最高気温: " + weatherMain.getTempMax(false) + "（元のデータ: " + weatherMain.getTempMax(true) + ")" + "\n" +
                        "最低気温: " + weatherMain.getTempMin(false) + "（元のデータ: " + weatherMain.getTempMin(true) + ")" + "\n" +
                        "湿度：　" + weatherMain.getHumidity() + "\n" +
                        "気圧：　" + weatherMain.getPressure() + "\n" +
                        "海面気圧： " + weatherMain.getSeaLevel() + "\n" +
                        "地点気圧：　" + weatherMain.getGrndLevel() + "\n" +
                        "----------" + "\n");
            }
        }
    }

}

File: C-Sharp_Weather_API_Library/WeatherCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static C_Sharp_Weather_API_Library.OpenWeatherAPI;
using static C_Sharp_Weather_API_Library.OpenWeatherAPI.WeatherList;

namespace C_Sharp_Weather_API_Library
{
    internal class WeatherCommand : CommandBase
    {

        void CommandBase.runCommand(string command)
        {
            if (command.Equals("get_weather"))
            {
                OpenWeatherAPI openWeatherAPI = OpenWeatherAPI.GetInstanceAsync().Result;
                WeatherList weatherList = openWeatherAPI.GetWeatherList[0];
                Weather weather = weatherList.GetWeather[0];
                WeatherMain weatherMain = weatherList.GetWeatherMain;

                Console.WriteLine("天気：" + weather.GetDescription + "\n" +
                    "最高気温：" + weatherMain.GetTempMax + "\n" +
                    "最低気温： " + weatherMain.GetTempMin + "\n");
            }
        }
    }
}
